### PR TITLE
[APPS-1411]: add polling to Explorer tables

### PR DESCRIPTION
**Pull request by plam-ml**
## Description 

- add polling for Txn, Epochs and Checkpoints
- https://mysten-labs.slack.com/archives/C0578KFD9D2/p1688670040245509
- https://mysten.atlassian.net/browse/APPS-1411

Screenshot 2023-07-06 at 2 57 49 PM

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: apps/explorer/src/components/Activity/index.tsx [hunk 1]
```diff
@@ -4,10 +4,13 @@
 import { useState } from 'react';
 // import toast from 'react-hot-toast';
 
+import { toast } from 'react-hot-toast';
+
 import { EpochsActivityTable } from './EpochsActivityTable';
 import { TransactionsActivityTable } from './TransactionsActivityTable';
 import { CheckpointsTable } from '../checkpoints/CheckpointsTable';
 // import { PlayPause } from '~/ui/PlayPause';
```
Comment (Jordan-Mysten) [inline apps/explorer/src/components/Activity/index.tsx:12]: Remove commented out one

File: apps/explorer/src/components/Activity/TransactionsActivityTable.tsx [hunk 2]
```diff
@@ -35,9 +36,10 @@ export function TransactionsActivityTable({
 		cacheTime: 24 * 60 * 60 * 1000,
 		staleTime: Infinity,
 		retry: false,
+		refetchInterval: 500,
```
[the remark this answers is not in the document]
Comment (plam-ml) [reply apps/explorer/src/components/Activity/TransactionsActivityTable.tsx:39]: oops, sorry it should be a passthrough

[the remark this answers is not in the document]
Comment (plam-ml) [reply apps/explorer/src/components/Activity/TransactionsActivityTable.tsx]: done

File: apps/explorer/src/components/Activity/index.tsx (hunk 1 above)
Comment (plam-ml) [reply apps/explorer/src/components/Activity/index.tsx]: done

**Review by Jordan-Mysten — APPROVED** [review]

**Comment by Jibz-Mysten** [comment]
@plam-ml one small thing, I think we shouldn't show the loading for refetch.

**Review by Jibz-Mysten — APPROVED** [review]

**Review by Jordan-Mysten — APPROVED** [review]

**Pull request merged**